protocol.md

```diff
@@ -1056,6 +1095,12 @@ export interface TextDocumentSyncOptions {
 
 interface ServerCapabilities {
 	/**
+	 * The server supports $/partialResult notifications.
+	 * If true, the client is allowed to let the result in the final response be null
+	 * and only send the result through $/partialResult
+	 */
+	streaming?: boolean;
```

Comment:
Ah, this only applies to requests sent by the server. I think it would help to clarify the documentation here (and above).